app/views/videos/show.html.haml

```diff
@@ -10,7 +10,8 @@
         %p= @video.description
         .actions
           %a.btn.btn-primary(href="") Watch Now
-          %a.btn.btn-default(href="") + My Queue
+          / %a.btn.btn-default(href="") + My Queue
```

Comment:
make sure you remove commented code. ESPECIALLY from your views. If you look in dev tools, you will see this line rendered as an html comment, for all to see.